installer/parse-ros-package-deps.py

```diff
@@ -31,11 +35,13 @@ def main():
     deps = doc.findall('exec_depend')
     dep_set |= {dep.text for dep in deps}
 
-    deps = doc.findall('test_depend')
-    dep_set |= {dep.text for dep in deps}
+    if os.getenv('TUE_INSTALL_TEST_DEPEND', 'false') == 'true':
```

Comment:
If this environment variable is being modified in tue-install-impl but not exported with the flag --doc-depend, will the `if` condition resolve correctly?

Reply:
See updated comment in the options of tue-install-impl.